subsys/bluetooth/host/gatt.c

```diff
@@ -754,6 +763,32 @@ static uint8_t gen_hash_m(const struct bt_gatt_attr *attr, uint16_t handle,
 	case BT_UUID_GATT_INCLUDE_VAL:
 	case BT_UUID_GATT_CHRC_VAL:
 	case BT_UUID_GATT_CEP_VAL:
+#if defined(CONFIG_BT_USE_PSA_API)
```

Comment:
please make a utility function. This applies to the whole PR. I really don't like ifdef soups.